frontend/src/components/infrastruktureinrichtung/InfrastruktureinrichtungVerortung.vue

```diff
@@ -150,7 +276,27 @@ export default class InfrastruktureinrichtungVerortung extends Mixins(
   }
 
   get stadtbezirke(): Array<StadtbezirkDto> {
-    return _.isNil(this.verortungModel) ? [] : _.sortBy(Array.from(this.verortungModel?.stadtbezirke), ["nummer"]);
+    return _.sortBy(_.toArray(this.verortungModel?.stadtbezirke), ["nummer"]);
+  }
+
+  get bezirksteile(): Array<BezirksteilDto> {
+    return _.sortBy(_.toArray(this.verortungModel?.bezirksteile), ["nummer"]);
+  }
+
+  get gemarkungen(): Array<GemarkungDto> {
+    return _.sortBy(_.toArray(this.verortungModel?.gemarkungen), ["nummer"]);
+  }
+
+  get kitaplanungsbereiche(): Array<KitaplanungsbereichDto> {
+    return _.sortBy(_.toArray(this.verortungModel?.kitaplanungsbereiche), ["nummer"]);
```

Comment:
Das mit `nummer` wird hier nicht funktionieren, da es das Attribute in Kitaplanungsbereich nicht gibt. Den Fehler hatte ich in einem vorherigen Commit eigentlich schon behoben gehabt

```suggestion
    return _.sortBy(_.toArray(this.verortungModel?.kitaplanungsbereiche), (k) => Number(k.kitaPlbT));
```

Reply:
Das Attribut gibt nun. Es wurde in der GeodatenEAI nicht gemappt. Nun wird es mitgeliefert.

Reply:
Berichtigt und anstelle "nummer" wird "kitaPlb" verwendet.